dist_linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash -e
+
+template_dir="dist-template/windows"
```

Comment:
Well, it looks like Windows binaries will be included in Linux distro and vice versa...